### DISCUSSION TITLE
frontend/cesium/20211103-testing-cesium-billboard-performance-with-chrome

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#944] [Announcements]
# frontend/cesium/20211103-testing-cesium-billboard-performance-with-chrome

输出本篇文章主要是为记录测试和学习的过程，毕竟好久没有水一篇正经文章了。。。

https://oncew.com/frontend/cesium/20211103-testing-cesium-billboard-performance-with-chrome.html

**Comment by cyanfish-x** [comment]
不错的文章
